apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -6176,4 +6185,36 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
             this.autoFillDeliverToWarehouseAddress(warehouse);
         }
     }
+
+    /**
+     * To get tax list and assign values to local variables
+     *
+     * @memberof ProformaInvoiceComponent
+     */
+    public getAllDiscounts(): void {
+        this.store.pipe(select(p => p.company.isGetTaxesSuccess), takeUntil(this.destroyed$)).subscribe(isGetTaxes => {
```

Comment:
avoid single character variable

Reply:
Existing code was moved to a method by @rohit-walkover